index.html

```diff
@@ -360,6 +360,37 @@ <h2>Skills</h2>
             </ul>
         </div>
     </section>
+    <!-- End Section part -->
+
+    <!-- Start Contact form -->
+
+    <div class="contact-form">
+        <div class="formbg" alt="form background"></div>
+        <div class="form-header">
+          <div class="form-header__title">
+                <h4>Contact me</h4>
+            </div>
+            <div class="form-header__text">
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                I’d love to help with it!</p>
+         </div>
+        </div>
+        <div class="inputs">
+            <form action="" class="form">
+              <input type="text" class="item1" name="name" id="name" placeholder="Lorem Ipsum" required maxlength="30" >
+              <input type="email" class="item1" name="email" id="email" placeholder="loramipsum@gmail.com" required pattern=".+@beststartupever.com">
+              <textarea name="message" id="message" class="item3" placeholder="Write your message here" maxlength="500"></textarea>
```

Comment:
Kindly add required attribute to textarea element like you added other inputs. In this way, users will not send empty messages to you.